Arrowgene.Ddon.Database/Files/Database/Script/schema_sqlite.sql

```diff
@@ -496,3 +496,16 @@ CREATE TABLE IF NOT EXISTS "ddon_bazaar_exhibition" (
     "expire"          DATETIME                          NOT NULL,
     FOREIGN KEY("character_id") REFERENCES "ddon_character"("character_id")
 );
+
+CREATE TABLE IF NOT EXISTS "ddon_reward_box" (
+	"uniq_reward_id"	INTEGER NOT NULL,
+	"character_common_id"	INTEGER NOT NULL,
+	"quest_id"	INTEGER NOT NULL,
+	"num_random_rewards"	INTEGER NOT NULL,
+	"random_reward0_index"	INTEGER NOT NULL,
+	"random_reward1_index"	INTEGER NOT NULL,
+	"random_reward2_index"	INTEGER NOT NULL,
+	"random_reward3_index"	INTEGER NOT NULL,
```

Comment:
Would it not be better for it to be its own separate table instead of having 4 numbered random reward fields? If it's guaranteed that there could never be more than 4 its fine, but if its an arbitrary limit it'd be better to support having more than 4 random rewards

Reply:
I asked our data researchers and they said we can only have 4. It is a lot easier to manage 1 record per box entry.

Reply:
Is it a limit in the game client? What happens if you return more than 4 in the packet

Reply:
I just tried to reward 5 and it only shows 3. I think the 4th would come from the course GP benefit.

![image](https://github.com/sebastian-heinz/Arrowgene.DragonsDogmaOnline/assets/3587094/e3010515-f65b-435f-bc29-29443ef42ede)

Reply:
Understood, thanks